go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/rudderlabs/rudder-server
 
-go 1.21
-
-toolchain go1.21.1
```

Comment:
Out of curiosity, is this intentional?

Reply:
actually no, go mod tidy was causing some issues so I was trying different things, we will add it back